exist-core/src/main/java/org/exist/test/runner/XMLTestRunner.java

```diff
@@ -142,7 +142,7 @@ private static XMLTestInfo extractTestInfo(final Path path, final Document doc)
 
     private static @Nullable String getIdValue(final Node test) {
         final String id = ((Element)test).getAttribute("id");
-        return id.isEmpty() ? null : id;
+        return id.isBlank() ? null : id;
```

Comment:
This isn't needed.

Reply:
It is an extra little bit I saw I got wrong earlier when changing attribute implementation to be compliant with w3c. I think it does not hurt to keep it in this PR but more people think it should not be here then I am fine moving it.

Reply:
@line-o This could actually cause an issue, in so far as it would hide errors in the test case implementation.

Reply:
so, instead of returning null you would say it is better to return the string containing only space characters?

Reply:
> so, instead of returning null you would say it is better to return the string containing only space characters

@line-o No. If there is a string that is only whitespace characters, then an error should be raised.